### PR TITLE
[fix][doc]Add docs for ThresholdShedder

### PR DESCRIPTION
**Pull request by 315157973**
### Motivation
Add docs for ThresholdShedder
https://github.com/apache/pulsar/pull/17456

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Review by Anonymitaet — APPROVED** [review]

File: site2/docs/administration-load-balance.md [hunk 1]
```diff
@@ -132,6 +132,18 @@ This strategy tends to shed the bundles if any broker's usage is above the confi
 
 For example, assume you have three brokers, the average broker usage of broker1 is 40%, the average broker usage of broker2 and broker3 is 10%, then the cluster average usage is 20% ((40% + 10% + 10%) / 3). If you set `loadBalancerBrokerThresholdShedderPercentage` to `10`, then only broker1's certain bundles get unloaded, because the average usage of broker1 is greater than the sum of the cluster average usage (20%) plus `loadBalancerBrokerThresholdShedderPercentage`(10%).
 
+However, in some special cases, the above default strategy may cause low-load machines to be idle forever.
```
Comment (momo-jun) [inline site2/docs/administration-load-balance.md:135]:
```suggestion
However, in some special cases, the above default strategy cannot leverage the resources of low-load or idle machines.
```

File: site2/docs/administration-load-balance.md [hunk 2]
```diff
@@ -132,6 +132,18 @@ This strategy tends to shed the bundles if any broker's usage is above the confi
...
+
+For example:
+
+There are 11 brokers, of which 10 are loaded at 80% and 1 is loaded at 0%.
+The average load is 80% * 10 / 11 = 72.73%, and the threshold to unload is 72.73% + 10% = 82.73%.
+Since 80% < 82.73%, unload will not be triggered, and there is one idle Broker with load of 0%.
+
+So, we add a new option `lowerBoundarySheddingEnabled`.
+If `lowerBoundarySheddingEnabled=true`, based on the `ThresholdShedder`, a leader will make the lower boundary judgment of the load.
+When `current usage < average usage - threshold` (e.g. 0% < 82.73% - 10%), the broker with the highest load will be triggered to unload.
```
Comment (momo-jun) [inline site2/docs/administration-load-balance.md:145]:
```suggestion
If `current usage` is less than `average usage - lower boundary load`, e.g., 0% < (82.73% - 10%), the broker with the highest load will be triggered to unload.
```

File: site2/docs/administration-load-balance.md [hunk 3]
```diff
@@ -132,6 +132,18 @@ This strategy tends to shed the bundles if any broker's usage is above the confi
...
+However, in some special cases, the above default strategy may cause low-load machines to be idle forever.
+
+For example:
+
+There are 11 brokers, of which 10 are loaded at 80% and 1 is loaded at 0%.
+The average load is 80% * 10 / 11 = 72.73%, and the threshold to unload is 72.73% + 10% = 82.73%.
+Since 80% < 82.73%, unload will not be triggered, and there is one idle Broker with load of 0%.
+
+So, we add a new option `lowerBoundarySheddingEnabled`.
+If `lowerBoundarySheddingEnabled=true`, based on the `ThresholdShedder`, a leader will make the lower boundary judgment of the load.
```
Comment (momo-jun) [inline site2/docs/administration-load-balance.md:144]:
```suggestion
When `lowerBoundarySheddingEnabled` is set to `true`, a leader decides the lower boundary of the load.
```

**Comment by momo-jun** [comment]
@315157973 thanks for adding the docs! 
Since it works on top of `ThresholdShedder`, does it make sense to move this newly added content a bit later ([starting from here](https://github.com/apache/pulsar/pull/18675/files#diff-28f860baebb586ef32c135e35e748de9be77675fe575018ce2e18d7f9405d660R168)), following the complete content of `ThresholdShedder`?

**Comment by 315157973** [comment]
> @315157973 thanks for adding the docs! Since it works on top of `ThresholdShedder`, does it make sense to move this newly added content a bit later ([starting from Row168](https://github.com/apache/pulsar/pull/18675/files#diff-28f860baebb586ef32c135e35e748de9be77675fe575018ce2e18d7f9405d660R168)), following the complete content of `ThresholdShedder`?

This part explain the load balancing strategy. If we divide them into two parts, the meaning will be incoherent

**Comment by momo-jun** [comment]
> > @315157973 thanks for adding the docs! Since it works on top of `ThresholdShedder`, does it make sense to move this newly added content a bit later ([starting from Row168](https://github.com/apache/pulsar/pull/18675/files#diff-28f860baebb586ef32c135e35e748de9be77675fe575018ce2e18d7f9405d660R168)), following the complete content of `ThresholdShedder`?
> 
> This part explain the load balancing strategy. If we divide them into two parts, the meaning will be incoherent

Also make sense. My thought - it breaks the coherent content of `ThresholdShedder` and also has a dependency on the configuration of `ThresholdShedder`, so it looks a bit advancing if we introduce it ahead of `ThresholdShedder`. It's not a broker anyway :)

**Review by momo-jun — APPROVED** [review]

**Pull request merged**